functions/utils/getRequest.ts

```diff
@@ -0,0 +1,33 @@
+import { MetaTagInjector } from '../components/metaTagInjector'
```

Comment:
Can you write tests for this file to ensure that things are called iff they should be?
When you think of your test cases, strive forfull code coverage: try to write a test for each branch of your conditionals.
To check that things are called iff they should be, use [.toHaveBeenCalled](https://jestjs.io/docs/expect#tohavebeencalled), and also install [jest-extended](https://www.npmjs.com/package/jest-extended) and use the [mock matchers](https://jest-extended.jestcommunity.dev/docs/matchers/mock/) (like .toHaveBeenCalledAfter).

iff == if and only if